src/transformers/training_args.py

```diff
@@ -1783,7 +1783,7 @@ def __post_init__(self):
             raise ValueError("warmup_steps must be either 0 or > 1")
 
         if isinstance(self.fsdp, bool):
-            self.fsdp = "full_shard" if self.fsdp else ""
+            self.fsdp = FSDPOption.FULL_SHARD if self.fsdp else ""
```

Comment:
From the logic below it looks like this should be a list 

```suggestion
            self.fsdp = [FSDPOption.FULL_SHARD] if self.fsdp else ""
```